include/graphics.h

```diff
@@ -1,15 +1,18 @@
+#pragma once
+
+#include "types.h"
+#include <array>
 #include <string>
 #include <vector>
-#include "types.h"
 
 namespace BitBorn {
 class Graphics {
   public:
     Graphics(int screenWidth, int screenHeight, float fieldOfView, float depth);
 
-    void renderFrame(fCoord25D playerPos, std::vector<int> mapDimensions, std::string map,
+    void renderFrame(fCoord25D playerPos, std::array<int, 2> mapDimensions, std::string map,
```

Comment:
Hum this might be a merge conflict. My PRs also do that.

Reply:
Yeah, I didn't realize that till after I pushed. It is a separate commit here though, instead of having it all in one commit like on your branch. So I think we should keep this commit, and fix any conflicts in yours.